key.js

```diff
@@ -269,9 +518,24 @@ const startAnimation = () => {
 startButton.addEventListener('click', () => {
     buttonPress()
     bool = !bool;
+    console.log(bool);
     if (bool) {
         startAnimation();
     } else {
         homeAnimation();
     }
 })
+
+function logic() {
+    // ...
+    // ...
+    gameModule.incrementGameLevel();
+}
+
+// gameModule
+// gameController
+
```

Comment:
- `gameModule` or `gameState`: Contains the core game state, and the way to make changes to the game state.  This means that all the game related data, something like `lives`, `levels`, etc will be part of this module.  Only data, and I mean strictly data is supposed be in this module.
- `viewModule` or `gameView`: Contains the core DOM elements, and the logic/methods to manipulate those DOM elements.  Any change to view should always be accompanied via this object.  Everything related to events, styling changes, element movement, etc, everything related to a views, belongs here, and only here.
- `controllerModule` or `gameController`: This is where most of the core game logic goes, that means this particular module requires both the `gameState` and `gameView` objects to ensure changes to game data and view whenever an action is done by the user.

Here's a skeleton for every module to get started:

```js
const gameState = (() => {
  // this part is already there above, you just need few changes w.r.t the game you're developing.
})();

const gameView = (() => {
  // this should contain everything that's just view related.
  // You can refer the following code to understand what I mean:
  // https://github.com/Ch-sriram/Budgety/blob/692095d3237cc92f63b234cb8f05c7ae554bbc8d/script/app.js#L185
})();

const gameController = ((state, view) => {
  // this is where the entire game logic will be.
  // Now to understand this part, please check the code here: 
  // https://github.com/Ch-sriram/Budgety/blob/692095d3237cc92f63b234cb8f05c7ae554bbc8d/script/app.js#L415
})(gameState, gameView);
```

The hard part about all of this is to understand what has been done in reference code, and try to fit it into the context and code here.  And quite frankly, understanding and reading others code is how you'll get better.

Don't hesitate to make any changes to this PR, the changes need not be perfect, be shameless about your changes, even if they're wrong.  The way you'll learn is to make mistakes (silly ones too!), and when I review them, you'll know where you went wrong.